source/c_dataexchange.rst

```diff
@@ -381,26 +445,42 @@ This example .ini creates (a) new nested asset(s) from each record of the shapef
 
 You can copy paste this code in your own .ini file and zip it together with the shapefile.
 
-Rasters
--------
+SUF-HYD
++++++++++
+
+SUF-HYD files can be imported manually, by uploading a file to https://demo.lizard.net/import/
 
-.. warning::
-   Text required!
+We currently do not support GWSW-Hyd yet.
+
+The description of SUF-HYD files can be found here: https://www.riool.net/documents/20182/557556/suf-hyd-gegevens%20stelsel-1996-2003.pdf/512c2708-0594-4227-998e-f9c51bec6a50 
 
 Data downloads
 ==============
 
-.. warning::
-   Text required!
-
 Rasters
 -------
 
-.. warning::
-   Text required!
+Download of rasters is possible but limited via the Lizard portal.
+The current limit is a 1 milion by 1 milion (10^9) pixels download.
+Only possible when you are zoomed in far enough, depending on the resolution of the specific raster.
+
+Select a raster from the datalayers menu to the right.
```

Comment:
Lijstje van maken?

Reply:
Dit is voor nu beschreven voor de Lizard portal zelf. Binnenkort (nu al volgens mij) is raster export via de Lizard Catalogue de beste manier om dit te gebruiken. Alleen moet die tekst nog geschreven worden. Dat zal komende week wel gebeuren, dus dan updaten we dit sowieso.